app/policies/organisation_policy.rb

```diff
@@ -0,0 +1,25 @@
+class OrganisationPolicy < BasePolicy
+
+  def index?
+    current_user.superadmin? || current_user.admin? || current_user.organisation_admin?
+  end
+
+  def can_assign?
+    return true if current_user.superadmin? || current_user.admin?
+    return current_user.organisation.subtree.pluck(:id).include?(record.id) if current_user.organisation_admin?
+
+    return false
+  end
+
+  class Scope < Scope
```

Comment:
It isn't clear to me which `Scope` this is inheriting from. Is it `BasePolicy::Scope` or `ApplicationPolicy::Scope`? If it had an explicit namespace that would make it clearer.